pkg/controller/nodeipam/ipam/cidr_allocator.go

```diff
@@ -116,6 +127,12 @@ func New(kubeClient clientset.Interface, cloud cloudprovider.Interface, nodeInfo
 	switch allocatorType {
 	case RangeAllocatorType:
 		return NewCIDRRangeAllocator(kubeClient, nodeInformer, allocatorParams, nodeList)
+	case MultiCIDRRangeAllocatorType:
```

Comment:
this also should be under the feature gate, I swear I saw the featuregate in one of the PRs but I can't find it now

Reply:
Done